rosdep/python.yaml

```
@@ -5465,6 +5465,17 @@ python-ujson:
     vivid:
       pip:
         packages: [ujson]
+python3-ujson:
+  debian:
+    '*': [python3-ujson]
+  fedora: [python3-ujson]
+  gentoo: [dev-python/ujson]
+  nixos: [pythonPackages.ujson]
```

Following the pattern of other Python 3 NixOS rules, I'm pretty sure this is how the rule should look:

```suggestion
  nixos: [python3Packages.ujson]
```